dali/pipeline/data/buffer.h

```diff
@@ -674,7 +674,10 @@ class DLL_PUBLIC Buffer {
   static double growth_factor_;
   static double shrink_threshold_;
 
-  static bool default_pinned();
+  static bool default_pinned() {
```

Comment:
Can't you just move `RestrictPinnedMemUsage` definition here instead of adding `default_pinned` on top of it?

Reply:
I tried.... and it didn't help :upside_down_face: